### DISCUSSION TITLE
New Paper: Online Mixed-Integer Optimization in Milliseconds

### DISCUSSION THREAD

**Discussion by adishavit** [cvxpy/cvxpy#1385] [Ideas]
There's a very interesting new paper out: [**Online Mixed-Integer Optimization in Milliseconds**](https://arxiv.org/abs/1907.02206)
Would be awesome to see it integrated into CVXPY.
[They actually use CVXPY](https://github.com/bstellato/mlopt)!

**Comment by akshayka** [comment]
@bstellato is actually one of CVXPY's four project maintainers.

I believe this might be the source code for the project: https://github.com/bstellato/mlopt

It looks like it may no longer be maintained?
